src/components/ConnectedList.jsx

```diff
@@ -31,14 +31,11 @@ class ConnectedList extends Component {
     const { t, connectors, children } = this.props
     return (
       <div className="content">
-        <div className="con-top-bar">
+        <div className="con-top-bar" data-tutorial="con-top-bar">
```

Comment:
I think that `con-` is a class prefix for "Connect" (remember _CozyConnect_ ? Good ol'time), so we may remove it.